diff --git a/node_modules/expo-config-plugin-ios-share-extension/build/writeShareExtensionFiles.js b/node_modules/expo-config-plugin-ios-share-extension/build/writeShareExtensionFiles.js
index 5f0e1db..19c8be5 100644
--- a/node_modules/expo-config-plugin-ios-share-extension/build/writeShareExtensionFiles.js
+++ b/node_modules/expo-config-plugin-ios-share-extension/build/writeShareExtensionFiles.js
@@ -210,8 +210,6 @@ class ShareViewController: UIViewController {
     private func redirectToHostApp(sharedURL: String) {
         var urlComponents = URLComponents()
         urlComponents.scheme = hostAppURLScheme
-        urlComponents.host = "share"
-        urlComponents.path = "/"
         urlComponents.queryItems = [
             URLQueryItem(name: "url", value: sharedURL),
         ]